quickwit/quickwit-janitor/src/actors/garbage_collector.rs

```diff
@@ -554,4 +577,11 @@ mod tests {
         assert_eq!(counters.num_failed_storage_resolution, 0);
         assert_eq!(counters.num_failed_gc_run_on_index, 1);
     }
+
+    fn get_first_split_state(states: Vec<SplitState>) -> SplitState {
```

Comment:
Or simply `states[0]`. Might as well remove this function.

Reply:
Done!